app/scripts/views/marketing_snippet_ios.js

```diff
@@ -85,22 +85,26 @@ define([
       MarketingSnippetView.prototype.initialize.call(this, options);
 
       this._language = options.language;
-      this._able = options.able;
+
+      // ** Will be needed when Fx iOS launches
+      //this._able = options.able;
```

Comment:
Do you know the timeline for when this code will be needed again? We could in theory use able to switch between the "ios coming soon" and "download fx for ios", but is it worth it?